packages/vanilla/src/components/button/button.stories.js

```diff
@@ -33,46 +48,82 @@ const setBtnFill = (color, fill) => {
   }
 }
 
-const Template = ({ variant, color, fill, label, disabled, type, ariaLabel }) => (
-  `
-    <button
-      class="${variant === 'default' ? 'cbp-btn' : 'cbp-btn-square'} ${setBtnFill(color, fill)}"
-      type=${type}
-      ${disabled ? "disabled='true'" : ''}
-      ${variant === 'default' ? '' : 'aria-label=' + ariaLabel}
-    >
-      <i class="fas fa-clipboard-check"></i>${variant === 'default' ? label : ''}
-    </button>
-  `
-);
+const Template = ({ element, variant, color, fill, label, disabled, type, ariaLabel }) => {
+  const el = (element == 'button') ? 'button' : 'a';
```

Comment:
Can we remove this variable from here and the stories below? doesn't look like it's being used since the ternary below is checking the element control value.

Reply:
Yeah, this is leftover from a previous technique where I was outputting the tag itself from a variable, but this was getting messy and unmaintainable.